File: 240914/사칙연산 함수/quadratic-operations-function.cpp
#include <iostream>
#include <string>
using namespace std;
int add(int a, int b) {
    return a + b;
}
int subtract(int a, int b) {
    return a - b;
}
int multiply(int a, int b) {
    return a * b;
}
int devide(int a, int b) {
    return a / b;
}

int main() {
    int a, c, result;
    char o;
    cin >> a >> o >> c;
    if (o == '+') {
        result = add(a, c);
        cout << a << " " << o << " " << c << " " << "= " << result;
    }
    else if (o == '-') {
        result = subtract(a, c);
        cout << a << " " << o << " " << c << " " << "= " << result;
    }
    else if (o == '*') {
        result = multiply(a, c);
        cout << a << " " << o << " " << c << " " << "= " << result;
    }
    else if (o == '/') {
        result = devide(a, c);
        cout << a << " " << o << " " << c << " " << "= " << result;
    }
    else {
        cout << "False";
    }
    return 0;
}

File: 240914/사칙연산 함수/quadratic-operations-function.py
def add(a, b):
    return a + b

def subtract(a, b):
    return a - b

def multiply(a, b):
    return a * b

def divide(a, b):
    return a // b 

a, o, c = input().split()
a = int(a)
c = int(c)

if o == '+':
    result = add(a, c)
    print(f"{a} {o} {c} = {result}")
elif o == '-':
    result = subtract(a, c)
    print(f"{a} {o} {c} = {result}")
elif o == '*':
    result = multiply(a, c)
    print(f"{a} {o} {c} = {result}")
elif o == '/':
    result = divide(a, c)
    print(f"{a} {o} {c} = {result}")
else:
    print("False")